UQL/Interpreter.php

```diff
@@ -281,7 +283,9 @@ private function parseValue($value)
     private function callFunction(ASTFunctionCall $functionCall)
     {
         try {
-            return $this->extensionContainer->callFunction($functionCall->getFunctionName(), $functionCall->getArguments());
+            $context = new Context($this->dataSource->getEntityClass());
```

Comment:
For now.
We started adding factory for this object, but it was yet another dependency for this class.
Definitely something to improve before final version.